src/components/utils/InputText.vue

```diff
@@ -144,4 +161,27 @@ export default Vue.component('input-text', {
     margin: 0;
 }
 
+input[type=text].u-form-input-plaintext,
+input[type=password].u-form-input-plaintext {
+    display: inline-block;
+    width: calc(100% - 35px);
+    border-bottom: 0;
+    padding-right: 0;
+}
+
+.u-input-plaintext {
+    display: inline-block;
+    line-height: 40px;
+    width: 30px;
+    text-align: center;
+    cursor: pointer;
+    opacity: 0.5;
+    transition: opacity 0.2s;
+}
+
+.u-input-plaintext.active,
+.u-input-plaintext:hover {
+    opacity: 1;
+}
+
```

Comment:
All of the above class names should be using the `u-input-text-` prefix, eg. `.u-input-text-plaintext`.